platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/HashSessionIdManager.java

```diff
@@ -47,13 +47,14 @@
 
 import org.codice.ddf.platform.util.RandomNumberGenerator;
 import org.eclipse.jetty.server.session.AbstractSession;
-import org.eclipse.jetty.server.session.AbstractSessionIdManager;
 
 /**
  * Custom implementation of the {@link org.eclipse.jetty.server.SessionIdManager} that shares session
  * data between sessions in a cluster.
  */
-public class HashSessionIdManager extends AbstractSessionIdManager {
+@edu.umd.cs.findbugs.annotations.SuppressWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "not changing")
+public class HashSessionIdManager extends org.eclipse.jetty.server.session.HashSessionIdManager {
+    //changed this to extend Jetty's impl instead of the abstract class because of a hack in pax web
```

Comment:
I'm a little confused, this seems to be going from a jetty abstract class to a jetty class. Where does the pax web hack come in?  Also, What's the difference between this class and the jetty version of HashSessionIdManager.  It seems we override all of its functions anyway, so the fact that we are using it over AbstractSessionIdManager doesn't change anything.  And that class seems to be the same as this class except this uses a concurrentHashMap and that class uses synchronized everywhere.